packages/story-editor/src/components/colorPicker/colorPicker.js

```diff
@@ -72,13 +72,20 @@ function ColorPicker({
   allowsSavedColors = false,
   onClose = () => {},
   changedStyle = 'background',
+  onDimensionChange = () => {},
```

Comment:
Does it need a default? Isn't it always set (except maybe for some tests)?

Reply:
Yep, it's not set in some tests. Probably doesn't harm, having a default.